apps/fluent-tester/src/FluentTester/TestComponents/ContextualMenu/ContextualMenuTest.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 import * as React from 'react';
-import { Text, View, Switch } from 'react-native';
+import { Text, View, Switch, Platform } from 'react-native';
```

Comment:
Why do we need `Platform`?

Reply:
line 121. Basically, custom fonts don't work on macOS yet, so I had to do a platform check in some test code